carsus/io/vald/vald.py

```diff
@@ -191,40 +249,42 @@ def extract_linelist(self, vald):
         -------
             pandas.DataFrame
                 vald linelist containing only the following columns:
-                atomic_number or chemical, ion_charge, wavelength, log_gf, rad, stark, waals
+                atomic_number or chemical, ion_charge, wavelength, e_low, log_gf, rad, stark, waals
+                optionally: v_mic (if stellar linelist) and e_up, j_lo, j_up (if not shortlist)
         """
-        if self.strip_molecules:
-            return vald[
-                [
-                    "atomic_number",
-                    "ion_charge",
-                    "wavelength",
-                    "log_gf",
-                    "e_low",
-                    "e_up",
-                    "j_lo",
-                    "j_up",
-                    "rad",
-                    "stark",
-                    "waals",
-                ]
-            ].copy()
+        if self.shortlist:
+            linelist_mask = [
+                "chemical",
+                "ion_charge",
+                "wavelength",
+                "log_gf",
+                "e_low",
+                "rad",
+                "stark",
+                "waals",
+            ]
+            if self._stellar_linelist:
```

Comment:
Please add a comment!

Reply:
It's actually already in the docstring above explaining. I think adding it here is redundant?